src/wasm2c.c.tmpl

```diff
@@ -27,7 +28,7 @@
        , ((t)table.data[x].func)(__VA_ARGS__))
 
 #define RANGE_CHECK(mem, offset, len) \
-  if (UNLIKELY(offset + (uint64_t)len > mem->size)) TRAP(OOB)
+  if (UNLIKELY(offset + (u64)len > mem->size)) TRAP(OOB)
```

Comment:
Is this change unrelated?  Maybe split into its own PR?

Reply:
This was in response to https://github.com/WebAssembly/wabt/pull/1877#discussion_r842318972 . Sure, let's split it off.